src/AuthModule/components/ChangPass/ChangPass.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useState } from "react";
+import { useForm, SubmitHandler } from "react-hook-form";
+import "react-toastify/dist/ReactToastify.css";
+import styleChangPass from "./ChangPass.module.css";
+import axios from "axios";
+import { useToast } from "../../../context/TostifyContext";
+import { ChangPassProps, FormValues } from "../../../interfaces/Auth";
+
+const ChangPass: React.FC<ChangPassProps> = ({ handleClose }) => {
+  const { showSuccessToast, showErrorToast } = useToast();
+  const [spinner, setSpinner] = useState<boolean>(false);
+  const [showPassword, setShowPassword] = useState<boolean>(false);
```

Comment:
Password with toggle functionality will be needed in multiple places, instead of repeating this logic into every place, We might consider creating a password input with toggle functionality.